microsoft-edge/devtools-guide-chromium/security/index.md

```diff
@@ -63,7 +63,7 @@ If you have not set up HTTPS on your server, [Let's Encrypt](https://letsencrypt
 
 ### Mixed content
 
-**Mixed content** means that the main origin of a page is secure, but the page requested resources from non-secure origins.  Mixed content pages are only partially protected because the HTTP content is accessible to sniffers and vulnerable to man-in-the-middle attacks.
+**Mixed content**<!--[mixed content](/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content)--> means that the main origin of a page is secure, but the page requested resources from non-secure origins.  Mixed content pages are only partially protected because the HTTP content is accessible to sniffers and vulnerable to man-in-the-middle attacks.
```

Comment:
I see you've done this a few times in this PR: removed a commented-out reference link from the bottom of the page, and added it as a commented-out inline link next to the corresponding word.

I don't know if the same concept applies here, but when I was doing development, we'd always delete commented-out code. Code that doesn't run is extremely unlikely to ever be used again, and keeping sections of it commented-out doesn't help anybody.
I tend to think that the same thing applies here. The article is likely to evolve over time and the link will remain commented-out.

I would suggest to either use it here (maybe it is useful for users to go and find out more about mixed content), or delete the comment entirely.
What do you think?

Reply:
Note that I understand that this PR is already a good cleanup of the remaining ref links. So it is 100% ok to not answer the above question in the scope of this PR. Land it as is, and then figure out what we want to do with the commented-out links.